phoenicis-tools/src/test/java/org/phoenicis/tools/archive/ExtractorTest.java

```diff
@@ -83,7 +89,12 @@ public void testUncompress_withSymbolicLinks() throws IOException {
     }
```

Comment:
I'm not sure the try/catch is needed here, because we could just add URISyntaxException to the throws in the method declaration like this:

```java
     @Test
     public void testUncompress_withSymbolicLinks() throws IOException, URISyntaxException {
         File  inputFile = new File(inputUrl.toURI().getPath(), "tarLink.tar.gz");

         final File temporaryDirectory = Files.createTempDir();
 
         temporaryDirectory.deleteOnExit();
 
         final List<File> extractedFiles = extractor.uncompress(inputFile, temporaryDirectory, e -> {});
 
         final File file1 = new File(temporaryDirectory, "file1.txt");
         final File file2 = new File(temporaryDirectory, "file1_link.txt");
 
         assertTrue(file1.exists());
         assertTrue(file2.exists());
 
         assertEquals("file1content", new String(FileUtils.readFileToByteArray(file1)));
         assertEquals("file1content", new String(FileUtils.readFileToByteArray(file2)));
 
         assertTrue(java.nio.file.Files.isSymbolicLink(Paths.get(file2.getPath())));
     }
```